pkg/controller/operators/olm/operator.go

```diff
@@ -158,6 +182,54 @@ func (a *Operator) syncDeployment(obj interface{}) (syncError error) {
 	return nil
 }
 
+func (a *Operator) syncOperatorGroups(obj interface{}) error {
+	op, ok := obj.(*v1alpha2.OperatorGroup)
+	if !ok {
+		log.Debugf("wrong type: %#v\n", obj)
+		return fmt.Errorf("casting OperatorGroup failed")
+	}
+
+	// write namespace matches to status field
+	selector, err := metav1.LabelSelectorAsSelector(&op.Spec.Selector)
+	if err != nil {
+		return err
+	}
+	operatorGroupOpts := metav1.ListOptions{LabelSelector: selector.String()}
+	namespaceList, err := a.OpClient.KubernetesInterface().CoreV1().Namespaces().List(operatorGroupOpts)
```

Comment:
I think we want to impersonate `op.Spec.ServiceAccount` when listing namespaces.

Reply:
We will in the near future, but for this PR I think we should stay away from the impersonation. We can get a lot out of the operatorgroup without implementing that just yet.